src/index.html

```diff
@@ -164,7 +164,7 @@
                 <div>
                     <p>
                         <!--<img src="img/Receive.png" alt="" width="48" height="48"> &nbsp;Receive your <a href="files/eva_submission_package.zip" target="_blank">submission pack</a>, which will include:-->
-                        Download your <a href="files/eva_submission_package.zip" target="_blank">submission pack</a>, which will include:
+                        <!--Download your <a href="files/eva_submission_package.zip" target="_blank" onClick="ga('send', 'event', { eventCategory: 'Download', eventAction: 'Click', eventLabel: 'eva_submission_package.zip'});">submission pack</a>, which will include:-->

```

Comment:
It seems the last commit was the one that introduced them. So it would mean taking it out from the history, undoing those specific changes and committing only the GA related ones in this branch. Then yes, the submission template changes would be placed in a new branch.